### DISCUSSION TITLE
Dependency on CADIMPORT module of COMSOL

### DISCUSSION THREAD

**Discussion by PALMS-gui** [wmglab-duke/ascent#21] [Q&A] [answered]
After designing my own cuff (my guess) from cuff primitives I see that the framework requires also this CADIMPORT module in addition to the base COMSOL. 
Could you explain what is this needed for and maybe if this can be avoided? Thanks

**Comment by thedannymarsh** [comment, marked as answer]
Hi PALMS-gui,

ASCENT should run with just the base COMSOL package. Could you provide some more info on where you are finding a need for the CAD import module?

Thanks!

**Reply by PALMS-gui** [reply]
It is hard to notice, when exactly during execution it fetches this toolbox, but I know that opening generated _debug_geom.mph_ in COMSOL GUI needs that. It is custom setup cuff+geometry, but the cuff was assembled from existing primitives only, namely these ones:
![afbeelding](https://user-images.githubusercontent.com/72300527/160869480-f4f418ad-afdd-477b-8ab2-bec52508e524.png)

**Reply by thedannymarsh** [reply]
Hmm, strange. We don't have a CAD import module license here, and nothing in debug_geom.mph should require it. Could you post a screenshot of the specific message that COMSOL is producing regarding the CAD import module? Thanks!

**Reply by PALMS-gui** [reply]
I think my colleague shared with you the geometry file of the cuff. Opening that one in COMSOL fetches this additional toolbox: https://www.comsol.com/cad-import-module . Strange if it is not the same for you.

**Reply by thedannymarsh** [reply]
Could you confirm what version of COMSOL you are running? The screenshot of the message regarding the CAD Import Module, if you can provide it, would also be helpful to identify the source of the import. Thanks!
